### PR TITLE
Add support for tokio processes

### PR DESCRIPTION
**Pull request by notpeelz**

### CODE REVIEW

File: src/tokio.rs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+use tokio::process::Command;
+use tokio_crate as tokio;
+
+use crate::*;
```
Comment (qwandor) [inline src/tokio.rs:4]: Style nit: I prefer not to have wildcard imports, please import the types you need explicitly.

**Review by qwandor — COMMENTED** [review]
Thanks for the PR! Just a minor style nit then I'm happy to merge this.

(You can ignore the coverage job failure, it looks like a tool issue.)

**Review by qwandor — APPROVED** [review]

File: src/tokio.rs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+use std::os::unix::prelude::RawFd;
+
+use tokio::process::Command;
+use tokio_crate as tokio;
+
+use crate::{map_fds, preserve_fds, validate_child_fds, FdMapping, FdMappingCollision};
```
[the remark this answers is not in the document]
Comment (qwandor) [reply src/tokio.rs:6]: That's fine.

**Comment by notpeelz** [comment]
Yeah, that would be appreciated. I have a project that depends on these changes.

**Comment by notpeelz** [comment]
Thanks!

**Pull request merged**